sdk/src/accounts/customizedCadenceBulkAccountLoader.ts

```diff
@@ -0,0 +1,150 @@
+import { BulkAccountLoader } from './bulkAccountLoader';
+import { Commitment, Connection, PublicKey } from '@solana/web3.js';
+
+export class CustomizedCadenceBulkAccountLoader extends BulkAccountLoader {
+	private customIntervalIds: Map<string, NodeJS.Timeout>;
+	private customPollingGroups: Map<number, Set<string>>;
+	private defaultPollingFrequency: number;
+
+	constructor(
+		connection: Connection,
+		commitment: Commitment,
+		defaultPollingFrequency: number
+	) {
+		super(connection, commitment, defaultPollingFrequency);
+		this.customIntervalIds = new Map();
+		this.customPollingGroups = new Map();
+		this.defaultPollingFrequency = defaultPollingFrequency;
+	}
+
+	private updateCustomPolling(frequency: number): void {
```

Comment:
i think its important that this method returns the `this.loadChunk` promise, so that consumers can have a choice of awaiting for a newly added account to be fetched

Reply:
Hmmmm it's a little tricky.. having trouble being able to return the promise in this method without potentially calling this every time we update frequency. I definitely don't want to do something that will increase the fetching too much... but maybe it's okay if we do a refetch when we change cadence? Wdyt?